inf-clojure.el

```diff
@@ -803,6 +816,7 @@ process buffer for a list of commands.)"
           (inf-clojure-mode)
           (set-syntax-table clojure-mode-syntax-table)
           (setq-local inf-clojure-repl-type repl-type)
+          (defvar-local inf-clojure--previous-response-ended-with-prompt nil)
```

Comment:
It seems to me this should where all the other `defvar-locals` are, not inside the mode definition itself.